packages/fiori/src/UserMenu.hbs

```diff
@@ -0,0 +1,93 @@
+<ui5-responsive-popover
+		id="{{_id}}-user-menu-rp"
+		class="ui5-pm-rp"
+		tabindex="-1"
+		placement="Bottom"
+		vertical-align="Bottom"
+		horizontal-align="End"
+		@ui5-close={{_handlePopoverAfterClose}}
+		prevent-initial-focus
+		accessible-name={{accessibleNameText}}
+		.open={{open}}
+		.opener={{opener}}
+>
+	{{#if _selectedAccount}}
+		<div class="ui5-pm-selected-account" slot="header">
+			<ui5-avatar size="L" @click="{{_handleAvatarClick}}" .initials="{{_selectedAccount._initials}}" fallback-icon="person-placeholder">
+				{{#if _selectedAccount.avatar}}
+					<img src={{_selectedAccount.avatar}}/>
+				{{/if}}
+				<ui5-tag slot="badge" wrapping-type="None" design="Set1" color-scheme="7" title="{{_editAvatarTooltip}}">
+					<ui5-icon slot="icon" name="edit"></ui5-icon>

```

Comment:
I think the edit icon is not imported, these kind of things will be improved with the tsx, we will import the icon in the template and use its default export and bind it to the name|icon properties of the Icon|Button